program/src/processor/set_authority.rs

```diff
@@ -13,9 +19,85 @@ use crate::instruction::{
 ///   1. `[s]` config_authority
 ///   2. `[]` new_authority
 pub fn process_set_authority(
-    _program_id: &Pubkey,
-    _ctx: Context<SetAuthorityAccounts>,
-    _authority_type: AuthorityType,
+    program_id: &Pubkey,
+    ctx: Context<SetAuthorityAccounts>,
+    authority_type: AuthorityType,
 ) -> ProgramResult {
+    // Accounts validation.
+
+    // 1. authority
+    // - must be a signer
+    // - must match the authority in the account (checked in the match statement below)
```

Comment:
The instruction has `account` first, then `authority` - in case you wanted to make sure these comments and the ordering of checks matched.

Reply:
I removed the numbering to not give the impression that it is the account order.

Reply:
Thanks. Clearly it got me on both processors!